Makefile

```diff
@@ -68,3 +68,5 @@ docker-build:
 
 docker-run:
 	@npm run docker
+
+.PHONY: build test
```

Comment:
> Should we add the rest of the commands to PHONY in case of name collisions?

We don't have any other targets with the same name as files, but I guess it doesn't hurt. Done.

> Hah, and I only now just noticed that @blowery added a typo urn ⚱ command.

That was too good to not keep it :)